### PR TITLE
Feature/base nextflow script

### PR DESCRIPTION
**Pull request by viklund**
Basic structure with some help message and fake generation of bamfiles.

### CODE REVIEW

File: src/main.nf [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env nextflow
+
+/*
+WGS Structural Variations Pipeline
+*/
+
+
+// 0. Preflight checks
```
Comment (samuell) [inline src/main.nf]: Heh ... I mis-read "pre-flight as pref-light" first, and was confused ... its a nitpick, but maybe can clarify anyway? :)

File: src/main.nf [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+if (params.help) {
+    usage_message()
+    exit 1
+}
+
+if (!params.bam) {
+    log.info 'ERROR: You need to specify a bam file, see --help for more information'
```
Comment (samuell) [inline src/main.nf]: Shouldn't errors be `log.error`, nor `log.info` ?

File: src/main.nf [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    exit 1
+}
+
+bamfile = file(params.bam)
+
+if (! bamfile.exists()) {
+    log.info "ERROR: The bamfile, '$params.bam', does not exist"
```
Comment (samuell) [inline src/main.nf]: Same here, isn't there an `log.error`?

File: src/main.nf [hunk 4]
```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env nextflow
+
+/*
+WGS Structural Variations Pipeline
```
Comment (samuell) [inline src/main.nf:4]: Add some author info and dates? ... so people know who to bug? :)

Comment (viklund) [inline src/main.nf:4]: Will do that, as soon as we decide who it is :)

**Pull request merged**